docs/mkdocs.yml

```diff
@@ -23,31 +23,53 @@ site_dir: 'generated'
 
 nav:
   - Welcome:
-      - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+      - Landing: index.md
+  - What is KubeStellar?:
+      - Overview: readme.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
-          - Examples: direct/examples.md
-          - Using existing hosting cluster: direct/hosting-cluster.md
-          - KubeStellar on K3D: direct/deploy-on-k3d.md
+  - Quick Start: direct/get-started.md
+  - User Guide:
+          - Basics: direct/usage-basics.md
+          - Prerequisites: direct/pre-reqs.md
+          - Quick Start: direct/get-started.md
+          - Setting up KubeStellar:
+              - Set Up Kubestellar Core Components: 
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - Via Helm Chart: direct/common-setup-core-chart.md
+              - Commission Workloads: direct/commission-workloads.md
+          - Installation & Usage Examples:
+              - Example Cluster Preparation: direct/hosting-cluster.md
+              - Sample scenarios: direct/examples.md
```

Comment:
This file does not just have sample scenarios. It also has setup; in particular, it points to both the step-by-step and core-chart ways to setup.

I think that what we should have, in total, is:
- One quickstart setup that is concrete and uses `kind` (no k3d, no OpenShift).
- General setup instructions. These can include worked examples of particular steps, such as "for example, here is a way to create a k3d cluster with an Ingress controller with SSL passthrough enabled: ...".
- A collection of scenarios that can be run on any KubeStellar setup that has a hosting cluster and two WECs. These scenarios are parameterized by the names of three kubeconfig contexts, rather than insisting on `kind-kubeflex`, `cluster1`, and `cluster2`. The scenarios doc starts with an explanation of its parameters.
- The quickstart setup ends with a reference to the scenarios doc and fully literal commands to set the scenario parameter variables.

Reply:
The quickstart is actually built in significant part by use of multiple partial or complete imports from other files, so it can render as a single page. It is by no means a finished thing.

Reply:
The inclusion relationships are based on an old organization, so I think that we are free to re-arrange as we see fit as we work out a better organization.